File: VS code for java/DAA program/linearSearch.cpp
#include <iostream>
#include <vector>

int linearSearch(const std::vector<int>& arr, int target) {
    int n = arr.size();
    for (int i = 0; i < n; i++) {
        if (arr[i] == target) {
            return i; // Return the index of the target element if found
        }
    }
    return -1; // Return -1 if the target element is not found in the array
}

int main() {
    std::vector<int> arr = {17, 11, 13, 6, 9, 5};
    int target = 6;

    int result = linearSearch(arr, target);

    if (result != -1) {
        std::cout << "Element " << target << " found at index " << result << std::endl;
    } else {
        std::cout << "Element " << target << " not found in the array." << std::endl;
    }

    return 0;
}

File: VS code for java/DAta Structures/Linear_Search.java
public class Linear_Search {

    public static void main(String[] args) {
        int []arr = {3,5,2,7,9,11};
        int item=9;
        System.out.println(Search(arr, item));
          
    }
    public static int Search(int[] arr, int item){
        for(int i=0; i<arr.length; i++){
            if(arr[i]==item){
                return i;
                
            }    
             
        }
    return -1;
    } 
}
